packages/js/src/ui/context/CountContext.tsx

```diff
@@ -38,7 +38,11 @@ export const CountProvider = (props: ParentProps) => {
     setUnreadCounts(newMap);
   };
 
-  onMount(updateUnreadCounts);
+  onMount(() => {
+    if (tabs().length > 0) {
+      updateUnreadCounts();
```

Comment:
Should we place the if statement in the `updateReadCounts` function and rename it to `updateTabCounts` so make it clearer?

Reply:
yes, @BiswaViraj please adjust to the suggestion